src/javascript/JContent/actions/deleteActions/Delete/Delete.jsx

```diff
@@ -28,8 +29,10 @@ const DeleteContent = ({data, onClose, isLoading, isMutationLoading, dialogType,
 
     let label;
     if (locked) {
-        label = t(`jcontent:label.contentManager.deleteAction.locked.${dialogType}.content`, {
-            name: firstNode?.displayName,
+        const labelChoice = count > 1 ? `${dialogType}.contents` : `${dialogType}.content`;
+        label = t(`jcontent:label.contentManager.deleteAction.locked.${labelChoice}`, {
+            count: count,
+            name: data.jcr.nodesByPath.map(n => getName(n)).join(', '),
```

Comment:
i18n has built-in plural logic. Should be able to implement with the following: 

```
label = t(`jcontent:label.contentManager.deleteAction.locked.content`, {
            count: count,
            name: data.jcr.nodesByPath.map(n => getName(n)).join(', '),
```

Then add in the language.json, change key to key_plural e.g. change `contents` to `content_plural`